torch/_inductor/cudagraph_utils.py

```diff
@@ -54,3 +54,35 @@ def check_for_mutation(
     else:
         has_mutation = len(compiled_graph.mutated_inputs) != 0
         return None if not has_mutation else default_msg
+
+
+def get_use_stack_trace(node) -> Optional[str]:
+    for use in node.users:
+        if stack_trace := use.meta.get("stack_trace", None):
+            return stack_trace
+    return None
+
+
+def check_multiple_devices(
```

Comment:
probably a nit but: maybe rename to `check_multiple_devices_or_any_cpu_nodes`? IIRC, you can have a graph that's 100% cpu (single-device) but we want this fn to trip and not use cudagraphs

Reply:
Sure will do